selfdrive/controls/lib/drive_helpers.py

```diff
@@ -211,3 +211,27 @@ def get_speed_error(modelV2: log.ModelDataV2, v_ego: float) -> float:
     vel_err = clip(modelV2.temporalPose.trans[0] - v_ego, -MAX_VEL_ERR, MAX_VEL_ERR)
     return float(vel_err)
   return 0.0
+
+
+class CanSignalRateCalculator:
+  """
+  Calculates the instantaneous rate of a can signal by using the counter
```

Comment:
can -> CAN like it is on the next line. make sure to stay consistent

Reply:
done